doc/chapters/gateways/index.rst

```diff
@@ -134,7 +134,10 @@ All the values can be substituted with the wildcard character ``*`` with the mea
 "direction" set to ``*`` will mean both incoming and outgoing, and a ``method`` set to ``*`` will
 match all method and signal names for the interface and object path specified.
 
-If a bus configuration is just an empty array it means all access to that bus will be blocked.
+If a bus configuration is just an empty array it means all access to that bus will be blocked. When
+the ``dbus-proxy`` is intercepts with D-Bus message, configuration list is filtered due to D-Bus
```

Comment:
"...the dbus-proxy intercepts a D-Bus message..." and "...the configuration list is filtered due to the D-Bus..."